src/styles/index.css

```diff
@@ -0,0 +1,12 @@
+body {
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, "PT Sans", "Segoe UI",
+        "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
```

Comment:
I wasn't sure if the rest of these fonts should have been taken out; I'm somewhat unfamiliar with style sheets

Reply:
Add Open Sans and Darker Grotesque as well, as seen in the Figma that I linked in the ticket